### ISSUE TITLE
[MdAppDrawer] Allow for dynamic `md-permanent` attribute with App Drawer

### ISSUE THREAD

**Issue by Soletiq** [vuematerial/vue-material#1678]
In my app, the left app drawer is permanent in desktop mode, but not permanent in tablet/mobile. Currently, the `md-permanent` attribute only accepts the values full, clipped, and card. But we only use 'full', which is applied in desktop mode. However, when the window is resized into a smaller mode, I need to remove this attribute entirely, which does not appear to be possible. Setting this attribute to false results in a Vue Warning, since the value can only be one of the 3 mentioned above.

My point is, perhaps we could add a possible value of 'not-permanent' to the attribute md-permanent to allow for apps that have more dynamic app drawers.

**Comment by VdustR** [comment]
App drawer mode will be fully reactive(with generating a new instance) if this PR merged.

**Comment by VdustR** [comment]
I think this might help you.

https://codepen.io/VdustR/pen/wmbZGB

**Comment by byander** [comment]
Hello,
I also would like to use the md-app-drawer as a permanent in desktop mode, using the option md-persistent="full". For me this mode is excelent to my app.
But if users use the app on their cell phone, for me isn't to use the option md-persistent it's the best. If there was a like false or null attribute to md-persistent it would be perfect.

I did a little hack and it's probably not a good practice. I installed the "vue-screen-size" library to get screen size and I take the width and check it with a default value, e.g. 600px. So, I inserted two md-app-drawer with v-if, one to desktop and other device mode.
Example
`<md-app-drawer :md-active.sync="menuVisible" md-persistent="full" v-if="this.$vssWidth > widthMobile">`.
